test-app/build-tools/jsparser/package.json

```diff
@@ -10,9 +10,10 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "babel-traverse": "6.26.0",
-    "babel-types": "6.26.0",
-    "babylon": "6.18.0",
+    "@babel/traverse": "~7.13.17",
+    "@babel/types": "~7.13.17",
+    "@babel/parser": "7.13.16",
+    "@babel/plugin-proposal-decorators":"7.13.15",
```

Comment:
Can we bump these to latest? `~7.14.x`

Reply:
I dont see any issue in bumping it. However it would require testing again. So maybe first merge this. The. We can locally update and test